packages/bundle-source/src/index.js

```diff
@@ -402,6 +402,7 @@ export default async function bundleSource(
   options = {},
   powers = undefined,
 ) {
+  assert(fs.existsSync(startFilename));
```

Comment:
un-requested feedback: I still would prefer avoiding `Sync` APIs where possible. In this case `stat` and checking for an `ENOENT` error is the alternative